src/Avalonia.X11/Clipboard/X11Clipboard.cs

```diff
@@ -317,9 +337,20 @@ public Task ClearAsync()
                     textFormatAtoms.Add(formatAtom);
                 }
                 else
+                {
                     formats.Add(format);
+
+                    if(!hasImage)
+                    {
+                        if (format.Identifier is "image/jpeg" or "image/jpg" or "image.png")
```

Comment:
Should be `image/png` not `image.png`. Please introduce and reuse constants to avoid typos :)